cypress/support/pages/home.pageObject.js

```diff
@@ -7,10 +7,24 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  get yourFeedLink() {
+    return cy.getByDataCy('your-feed-link');
+  }
+
   assertHeaderContainUsername(username) {
```

Comment:
The method `assertHeaderContainUsername` seems to have a grammatical error. It should likely be `assertHeaderContainsUsername` to correctly reflect the action being performed.